src/transform/ThemeTransform.css

```diff
@@ -46,15 +46,16 @@ styles with only the things which need tweaking.
 }
 
 /* baseline div, span, ul, li */
-.pagelib-theme-dark .content div,
+/* en > Pantone > 792312384 */
+.pagelib-theme-dark .content :not(.color_swatch) > div,
```

Comment:
I don't mind merging this for a quick fix since you've tested so many articles for regressions but I think this problem would be better solved and less fragile with the classification techniques used in ThemeTransform.classifyElements(). There's some discussion on the trouble with not-ing ancestors [here](https://stackoverflow.com/a/7084147/970346).